scripts/index.js

```diff
@@ -0,0 +1,79 @@
+const up = document.getElementById("up")
```

Comment:
This setup works, but if the script tag was moved from the bottom of the web page to the top, they would stop working, because they would attempt to run before the html loads.  I would recommend grabbing these elements in the functions that need them, rather than doing it all at the same time here.